core/client/src/main/java/alluxio/client/netty/NettyUnderFileSystemBlockReader.java

```diff
@@ -137,32 +124,25 @@ public ByteBuffer read(InetSocketAddress address, long ufsFileId, long offset, l
   /**
    * {@inheritDoc}
    *
-   * Releases the underlying buffer of previous/current read response.
+   * Release the underlying buffer of previous/current read response.
    */
   @Override
-  public void close() {
-    cleanup();
-  }
-
-  /**
-   * Releases the underlying buffer of previous/current read response.
-   */
-  private void cleanup() {
+  public void close() throws IOException {
     if (mReadResponse != null) {
       mReadResponse.getPayloadDataBuffer().release();
       mReadResponse = null;
     }
   }
 
   /**
-   * Class that contains metrics about {@link NettyUnderFileSystemFileReader}.
+   * Class that contains metrics about {@link NettyUnderFileSystemBlockReader}.
    */
   @ThreadSafe
   private static final class Metrics {
-    private static final Counter NETTY_UFS_READ_OPS =
-        MetricsSystem.clientCounter("NettyUfsReadOps");
-    private static final Counter NETTY_UFS_READ_FAILURES =
-        MetricsSystem.clientCounter("NettyUfsReadFailures");
+    private static final Counter NETTY_UFS_BLOCK_READ_OPS =
+        MetricsSystem.clientCounter("NettyUFSBlockReadOps");
```

Comment:
I think the metrics counter name (the string) uses "UFS". I am fine with using "Ufs" if that looks more consistent. your call